hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/MaintainIndexService.kt

```diff
@@ -44,9 +44,10 @@ class MaintainIndexService(
   }
 
   private fun doPrepareIndexForRebuild(indexStatus: IndexStatus): IndexStatus {
-    indexStatusService.markBuildInProgress()
+    indexStatusService.markBuildAbsent()
```

Comment:
When the build is in progress then queue listeners can process messages and write prisoners to the index.
We want to hold off doing that until the index has been created properly

Reply:
Not sure if we need a test for this behaviour or how hard that would be?